### PR TITLE
Open source styling updates

### PR DESCRIPTION
**Pull request by omonk**
Updates to all tickets below

[why use open source](https://trello.com/c/6bf4cfEm/484-why-use-open-source-component)
- Padding update to bottom of section

[upcoming events](https://trello.com/c/tLZy8Bj4/530-upcoming-events-section-for-os-page )
- Align title and event list
- Add description
- Change title
- Remove cta as requested by James

[github deliverables](https://trello.com/c/rS48DfwP/518-github-repo-related-work-open-deliverables-contributions-sections-lambda)
- Change spacing between CTA
- Update colour of repo text and HR

### CODE REVIEW

File: src/components/Common/seoLinks.js [hunk 1]
```diff
@@ -21,19 +22,19 @@ export default function SeoLinks({ items, ...props }) {
     <PagePaths
       render={pathsById => (
         <List as="ul" {...props}>
-          {(items || []).map((item, i) => {
+          {(items || []).map(item => {
             const path = pathsById[item.id]
             if (path) {
```
Comment (GBouffard) [inline src/components/Common/seoLinks.js]: I'd replace by `{path && <ListItem key={generate()}>` for nicer refactoring reasons

File: src/components/Common/seoLinksContainer.js [hunk 2]
```diff
@@ -15,8 +16,8 @@ const StyledLinksColumn = styled.div`
   `}
 `
 
-const SeoLinksColumn = ({ speciality: { title, items } }, index) => (
-  <Col width={[1, 1, 1, 1, 1 / 2, 3 / 12]} key={index}>
+const SeoLinksColumn = ({ speciality: { title, items }, index }) => (
```
Comment (GBouffard) [inline src/components/Common/seoLinksContainer.js]: Good call. I didnt see that I missed that 👍

File: src/components/OpenSource/WhyOpenSource.js [hunk 3]
```diff
@@ -1,13 +1,21 @@
...
-
+import breakpoint from 'styled-components-breakpoint'
 import { Grid, Row, Col } from '../grid'
 import TitleAndList from '../Common/TitleAndList'
 import Subtitle from '../Typography/Subtitle'
 import LogoGrid from '../Common/LogoGrid'
 
 const StyledGrid = styled(Grid)`
-  padding-bottom: ${props => props.theme.space[6]};
+  padding-bottom: ${props => props.theme.space[4]};
```
Comment (GBouffard) [inline src/components/OpenSource/WhyOpenSource.js]: Maybe all the CSS as `${({ theme }) => theme.spacing[4]}` for consistency. It's not a must do.

File: src/components/Common/Repo/Repo.js [hunk 4]
```diff
@@ -44,12 +50,14 @@ const Repo = ({
           <Ellipsis>{nameWithOwner}</Ellipsis>
         </Subtitle>
       </ConditionalLink>
-      <BodyStylised small={isImage} noPadding>
-        <Ellipsis>{pullRequestCount} Contributions</Ellipsis>
-      </BodyStylised>
-      <BodyStylised small={isImage} noPadding>
+      <Ellipsis>
+        <StyledBodyStylised small={isImage} noPadding>
+          {pullRequestCount} Contributions
```
[the remark this answers is not in the document]
Comment (omonk) [reply src/components/Common/Repo/Repo.js]: aha yeah I think we'll go for those big contributions metrics 😆

File: src/components/Common/seoLinksContainer.js (hunk 2 above)
Comment (omonk) [reply src/components/Common/seoLinksContainer.js]: a few errors in the console, easy to miss!

File: src/components/OpenSource/WhyOpenSource.js (hunk 3 above)
Comment (omonk) [reply src/components/OpenSource/WhyOpenSource.js]: I was using the destructuring logic yesterday as I wanted to get another prop for the background colour in the section wrapper. I reckon using `${(props) => props.spacing....}` is still ok. happy to mix them as its total preference.

File: src/components/Common/Events.js [hunk 5]
```diff
@@ -29,22 +17,36 @@ const getAllUpcomingEvents = events =>
     .sort((a, b) => (a.date <= b.date ? -1 : 1))
     .slice(0, 5)
 
-const EventSection = ({ events, title, eventIcon }) => {
+const EventSection = ({ events, title, description, eventIcon }) => {
   const displayedEvents = events.length ? getAllUpcomingEvents(events) : []
```
[the remark this answers is not in the document]
Comment (omonk) [reply src/components/Common/Events.js]: ah you're right, will remove

File: src/components/Common/seoLinks.js (hunk 1 above)
Comment (omonk) [reply src/components/Common/seoLinks.js]: changed this for less repetition

**Pull request merged**